### DISCUSSION TITLE
Unable to install open3d

### DISCUSSION THREAD

**Discussion by eduardo4jesus** [isl-org/Open3D#6590] [General]
I am unable to install open3d using pip.

```
(open3d) edreis@compute-0-1:~$pip -v  install open3d
Using pip 23.3.2 from /home/edreis/miniconda3/envs/open3d/lib/python3.10/site-packages/pip (python 3.10)
ERROR: Could not find a version that satisfies the requirement open3d (from versions: none)
ERROR: No matching distribution found for open3d
(open3d) edreis@compute-0-1:~$
```

I have also tried to install `libomp` as mentioned by @stockdam in https://github.com/isl-org/Open3D/discussions/6284#discussioncomment-7680194_, and by  @cwreynolds in https://github.com/isl-org/Open3D/discussions/6032#discussioncomment-5433725; but that it also didn't help.

I also tried `open3d-cpu` and even older version, and nothing.

Does anyone have a solution or any hints on this issue?
Cheers,

**Comment by EwingKang** [comment]
On my Intel I7+Ubuntu22.04+Python 3.10, the command `pip -v install open3d` ran just fine with Open3D v0.18.0. What OS and CPU are you using?

**Reply by eduardo4jesus** [reply]
I tried on my personal machine inside a `conda` environment: 
```
(open3d) eduardoj@WorkSmart:~/Code/eduardo4jesus$ lscpu | grep "Model name"
Model name:                         11th Gen Intel(R) Core(TM) i7-11370H @ 3.30GHz
(open3d) eduardoj@WorkSmart:~/Code/eduardo4jesus$ uname -a
Linux WorkSmart 6.5.0-14-generic #14~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Mon Nov 20 18:15:30 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
(open3d) eduardoj@WorkSmart:~/Code/eduardo4jesus$ pip --version
pip 23.3 from /home/eduardoj/miniconda3/envs/open3d/lib/python3.11/site-packages/pip (python 3.11)
(open3d) eduardoj@WorkSmart:~/Code/eduardo4jesus$ 
```

and, I also tried on a remote server I work:
```
(open3d) edreis@compute-0-1:~$uname -a
Linux compute-0-1.local 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
(open3d) edreis@compute-0-1:~$lscpu | grep "Model name"
Model name:            Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz
(open3d) edreis@compute-0-1:~$pip --version
pip 23.3.2 from /home/edreis/miniconda3/envs/open3d/lib/python3.10/site-packages/pip (python 3.10)
```

**Comment by iolpltuisciaoln** [comment]
❯ pip -v  install open3d
Using pip 23.3.2 from /home/user/.local/lib/python3.12/site-packages/pip (python 3.12)
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement open3d (from versions: none)
ERROR: No matching distribution found for open3d

**Reply by johnthagen** [reply]
- #6433

**Comment by ssheorey** [comment]
@iolpltuisciaoln Sorry Python 3.12 isn't supported yet. Please follow the issue for Python 3.12 support.

@eduardo4jesus Open3D works fine for Ubuntu 22.04 on an Intel CPU. THere's perhaps something wrong with your environment -can you install other packages such as numpy?

**Reply by eduardo4jesus** [reply]
@ssheorey, I was able to install other packages.

```
eduardoj@WorkSmart:~$ conda activate open3d
(open3d) eduardoj@WorkSmart:~$ pip install numpy
Requirement already satisfied: numpy in ./miniconda3/envs/open3d/lib/python3.11/site-packages (1.26.2)
(open3d) eduardoj@WorkSmart:~$ pip install scipy
Collecting scipy
  Using cached scipy-1.11.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (60 kB)
Requirement already satisfied: numpy<1.28.0,>=1.21.6 in ./miniconda3/envs/open3d/lib/python3.11/site-packages (from scipy) (1.26.2)
Using cached scipy-1.11.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (36.4 MB)
Installing collected packages: scipy
Successfully installed scipy-1.11.4
(open3d) eduardoj@WorkSmart:~$ 
```

And, I just tried to install open3d again and it just worked on my machine. But it did not work on the remove machine I work on. So, I am still having this problem. :thinking:
